modules/apps/29-fee/keeper/keeper_test.go

```diff
@@ -127,19 +127,71 @@ func (suite *KeeperTestSuite) TestIsLocked() {
 	suite.Require().True(suite.chainA.GetSimApp().IBCFeeKeeper.IsLocked(ctx))
 }
 
-func (suite *KeeperTestSuite) TestDisableAllChannels() {
-	suite.chainA.GetSimApp().IBCFeeKeeper.SetFeeEnabled(suite.chainA.GetContext(), "port1", "channel1")
-	suite.chainA.GetSimApp().IBCFeeKeeper.SetFeeEnabled(suite.chainA.GetContext(), "port2", "channel2")
-	suite.chainA.GetSimApp().IBCFeeKeeper.SetFeeEnabled(suite.chainA.GetContext(), "port3", "channel3")
-
-	suite.chainA.GetSimApp().IBCFeeKeeper.DisableAllChannels(suite.chainA.GetContext())
-
-	suite.Require().False(suite.chainA.GetSimApp().IBCFeeKeeper.IsFeeEnabled(suite.chainA.GetContext(), "port1", "channel1"),
-		"fee is still enabled on channel-1 after DisableAllChannels call")
-	suite.Require().False(suite.chainA.GetSimApp().IBCFeeKeeper.IsFeeEnabled(suite.chainA.GetContext(), "port2", "channel2"),
-		"fee is still enabled on channel-2 after DisableAllChannels call")
-	suite.Require().False(suite.chainA.GetSimApp().IBCFeeKeeper.IsFeeEnabled(suite.chainA.GetContext(), "port3", "channel3"),
-		"fee is still enabled on channel-3 after DisableAllChannels call")
+func (suite *KeeperTestSuite) TestGetIdentifiedPacketFeesForChannel() {
+	suite.coordinator.Setup(suite.path)
+
+	// escrow a fee
+	refundAcc := suite.chainA.SenderAccount.GetAddress()
+	packetID1 := channeltypes.NewPacketId(suite.path.EndpointA.ChannelConfig.PortID, suite.path.EndpointA.ChannelID, 1)
+	packetID2 := channeltypes.NewPacketId(suite.path.EndpointA.ChannelConfig.PortID, suite.path.EndpointA.ChannelID, 2)
+	packetID5 := channeltypes.NewPacketId(suite.path.EndpointA.ChannelConfig.PortID, suite.path.EndpointA.ChannelID, 51)
+	fee := types.Fee{
+		AckFee:     defaultAckFee,
+		RecvFee:    defaultReceiveFee,
+		TimeoutFee: defaultTimeoutFee,
+	}
```

Comment:
nit: could use `types.NewFee(defaultRecvFee, defaultAckFee, defaultTimeoutFee)`

feel free to ignore